PointArray targets
PointArray starting_positions
int32 timeout_ms
---
PointArray[] global_plan